Lib/test/test_zipimport.py

```diff
@@ -40,7 +40,7 @@ def make_pyc(co, mtime, size):
         else:
             mtime = int(-0x100000000 + int(mtime))
     pyc = (importlib.util.MAGIC_NUMBER +
```

Comment:
I think the above few lines can be removed.

Reply:
the lines from 38->41?

Reply:
Yes. They are roughly equivalent to `mtime = int(mtime) & 0xFFFF_FFFF` if use an unsigned integer format.